courses/free-courses-te.md

```diff
@@ -1,39 +1,222 @@
+* [JAVA SCRIPT FOR BEGINNERS IN 7 HOURS || LEARN JAVA SCRIPT IN 7 HOURS || JAVA SCRIPT](https://www.youtube.com/watch?v=BTuCzffKh8E) - Sundeep Saradhi Kanthety
+* [JavaScript in Telugu || JavaScript in Telugu by Kotha Abhishek](https://www.youtube.com/playlist?list=PLv_sM9ZH4RUW_Pgz-6B0Q-YTfWvC7RVFN) - Chintu Tutorials
+
+
+### Laravel
+
+* [Laravel](https://www.youtube.com/playlist?list=PLYnehuuSeAHvBW7ruB1sPomY1SK_3fvx0) - Know something!!!
+
+
+### Machine Learning
+
+* [Machine Learning full course 6 Hours in telugu](https://www.youtube.com/watch?v=UehuI1w10lg) - Python Life
+* [Machine Learning in Telugu](https://www.youtube.com/playlist?list=PLVG0Zju2HPJe0bhmV6l1MEE-6h0MG-20P) - Nerchuko
+
 
+### MongoDB
 
-### Javascript
+* [Mongo db full course in 1hour in telugu](https://www.youtube.com/watch?v=ZQuQ-wHuPlg) - Python Life
+* [MongoDB Database Tutorials for beginners in Telugu](https://www.youtube.com/playlist?list=PLeCpoxUq3Ehk9FDCQvswVhg5qrODqJ4Xp) - Sai Gopi
 
-* [Full Java Script Tutorials in Telugu - Telugu Web Guru](https://www.youtube.com/playlist?list=PLh6Yk2rpZu2KqDjTuU_qHr-tI_CHOkIsN) - telugu web guru (YouTube)
+
+### NextJS
+
+* [Next.js Crash Course in Telugu | Next.js in Telugu](https://www.youtube.com/watch?v=yqJlmkgroik) - Telugu Skillhub
+
+
+### NodeJS
+
+* [NodeJS Tutorial](https://www.youtube.com/watch?v=MY2Vxtfn5Tw) - Telugu Skillhub
+* [Node JS in Telugu | Node.js Tutorial for Beginners in Telugu](https://www.youtube.com/playlist?list=PLYnehuuSeAHtu27M2By66v6kJpF_oDR5I) - Know something!!!
+
+
+### PHP
+
+* [PHP Development Tutorial in Telugu - Learn in 24 Hours Part 1](https://www.youtube.com/watch?v=sX6g3zyPXkA) - Telugu Computer World
+* [PHP Development Tutorial in Telugu - Learn in 24 Hours Part 2](https://www.youtube.com/watch?v=zZ0QNRMxWkE) - Telugu Computer World
+* [PHP Tutorials](https://www.youtube.com/playlist?list=PLC2mgeYbYNm8TRhX27z6JG0DtR8FR_WuO) - Telugu Computer World
 
 
 ### Python
 
-* [Python in Telugu - Step by Step Tutorials](https://www.youtube.com/playlist?list=PLC2mgeYbYNm-3aTUq98pbmrA3P1_m-aJR) - Telugu Computer World (YouTube)
+* [Python in Telugu For Beginners - Complete Tutorial in 10 Hours](https://www.youtube.com/watch?v=fP9IvI4qu80) - Telugu Computer World
+* [Full Python Tutorial in Telugu | Telugu Web Guru](https://www.youtube.com/playlist?list=PLh6Yk2rpZu2JgeekeyLRQcwZsfLNbW8zQ) - Telugu Web Guru
 
 
 ### R
 
-* [R Programming Language Training Videos In telugu - R ట్రైనింగ్ వీడియోస్ ఇన్ తెలుగు R programming ట్రైనింగ్ వీడియోస్ ఇన్ తెలుగు](https://www.youtube.com/playlist?list=PLXx2-0oYp1LO9H8ciGQaTr6SN80dteTlc) - VLR Training (YouTube)
+* [R Programming Language Training Videos In telugu - R ట్రైనింగ్ వీడియోస్ ఇన్ తెలుగు R programming ట్రైనింగ్ వీడియోస్ ఇన్ తెలుగు](https://www.youtube.com/playlist?list=PLXx2-0oYp1LO9H8ciGQaTr6SN80dteTlc) - VLR Training
+* [What is R programming in telugu](https://www.youtube.com/watch?v=02CTDv02awE) - Python Life
```

Comment:
This is only for 10 mins. We dont allow tutorials < 1hr in length. Please remove